internal/pkg/service/buffer/sink/tablesink/storage/repository/file_repo.go

```diff
@@ -128,9 +161,9 @@ func (r *FileRepository) get(k storage.FileKey) op.ForType[*op.KeyValueT[storage
 // The entity is stored in 2 copies, under "All" prefix and "InLevel" prefix.
 // - "All" prefix is used for classic CRUD operations.
 // - "InLevel" prefix is used for effective watching of the storage level.
-func (r *FileRepository) put(v storage.File, create bool) *op.TxnOp {
-	level := v.State.Level()
-	etcdKey := r.schema.AllLevels().ByKey(v.FileKey)
+func (r *FileRepository) put(oldValue, newValue storage.File, create bool) *op.TxnOp {
+	level := newValue.State.Level()
+	etcdKey := r.schema.AllLevels().ByKey(newValue.FileKey)
```

Comment:
I need to use the put method again for the new feature.

So I added `oldValue` here as well, as it is in the slice repo and as you recommended in a previous PR.